README.md

```diff
@@ -11,30 +11,60 @@
 **This is the README for the `2.x` version of Redoc (React-based).** 
 **The README for the `1.x` version is on the [v1.x](https://github.com/Redocly/redoc/tree/v1.x) branch**
 
+## About Redoc
+
+Redoc is an open-source tool for generating OpenAPI(aka Swagger) definitions. By default
```

Comment:
```suggestion
Redoc is an open-source tool for generating OpenAPI (aka Swagger) definitions. By default
```

Reply:
OpenAPI is not the same as Swagger, though. I think you could go around this by listing spec versions Redoc works with: OpenAPI 2.0 (Swagger), OpenAPI 3.0, OpenAPI 3.1.

Perhaps more importantly, Redoc doesn't generate API definitions - the H1 on this page is correct, it generates docs _from_ API definitions.

Reply:
I think it is the same thing as Swagger, especially in the common lexicon. However, it is "fka" and not "aka". 
```suggestion
Redoc is an open-source tool for generating OpenAPI (fka Swagger) definitions. By default
```

Reply:
I still think we would benefit from listing specific OAS versions that Redoc works with :thinking:

Reply:
ah, excellent points y'all - will fix!